src/Search/Api/Elasticsearch/Command/resources/labs-experiment.yaml

```diff
@@ -66,6 +66,3 @@ labs-experiment:
       type: string
     number:
       type: long
-    type:
```

Comment:
Yes its in the `_default_.yml` file which is shared across all the indexes, just removing to avoid it causing issues